btclightclient/btc_light_client_store.go

```diff
@@ -48,6 +49,11 @@ func NewMemStore() *MemStore {
 	}
 }
 
+func (s *MemStore) IsBlockPresent(h chainhash.Hash) bool {
```

Comment:
I think we can use this logic in the light client file. We don't need to add one more interface for this. 
However, if you see this is cleaner, please let me know. I am good at all points.